traitsui/qt4/tree_editor.py

```diff
@@ -1072,13 +1072,10 @@ def _new_actions(self, node, object):
                     name = add_node.get_name(object)
```

Comment:
This is a nit-picky comment but as long as we are refactoring code in this PR, how about refactoring the `_new_actions` method to return early if `len(add) == 0`.

```
# return early if there are no items to be added in the tree
if len(add) == 0:
    return items

for klass in add:
    ....
```

Refactoring the code like above makes the code slightly more readable by removing one level of indentation.

Similarly, we can change `if add_node is not None` to `if add_node is None` to remove one more level of indentation in the code.